src/Conf.php

```diff
@@ -29,9 +29,9 @@ class Conf
     /**
      * Create new instance of Conf class.
      */
-    public function __construct()
+    public function __construct($filepath)
     {
-        $this->file = storage_path('app/conf.json');
+        $this->file = ($filepath) ? $filepath : storage_path('app/conf.json');
```

Comment:
Sorry for keep asking you to make changes but even with the added setter there is a problem when trying to get the configuration, because when I'm trying to get the configuration it searches on the default configuration path... 
So it works saving in a custom path, but I cannot get the configuration from the custom path...